holoviews/operation/datashader.py

```diff
@@ -68,74 +69,91 @@ def dataset_pipeline(dataset, schema, canvas, glyph, summary):
 
 
 class Aggregate(ElementOperation):
+    """
+    Aggregate implements 2D binning for any valid HoloViews Element
+    type using datashader. By default it will simply count the number
+    of values in each bin but custom aggregators can be supplied
+    implementing mean, max, min and other reduction operations.
+
+    The bins of the aggregate are defined by the width and height and
+    the x_range and y_range. If x_sampling or y_sampling are supplied
+    the operation will ensure that a bin is no smaller than the
+    minimum sampling distance.
+    """
 
     aggregator = param.ClassSelector(class_=ds.reductions.Reduction,
                                      default=ds.count())
 
-    height = param.Integer(default=800)
+    height = param.Integer(default=800, doc="""
+       The height of the aggregated image in pixels.""")
 
-    width = param.Integer(default=600)
+    width = param.Integer(default=600, doc="""
+       The width of the aggregated image in pixels.""")
 
```

Comment:
I thought one could only get the overall size of the plot, not the size of the data area?  Anyway, either is better than nothing!